hikari/impl/cache.py

```diff
@@ -305,6 +309,130 @@ def update_emoji(
         self.set_emoji(emoji)
         return cached_emoji, self.get_emoji(emoji.id)
 
+    def _build_sticker(
+        self,
+        sticker_data: cache_utility.GuildStickerData,
+    ) -> stickers.GuildSticker:
+        return sticker_data.build_entity(self._app)
+
+    def clear_stickers(self) -> cache.CacheView[snowflakes.Snowflake, stickers.GuildSticker]:
+        if not self._is_cache_enabled_for(config_api.CacheComponents.GUILD_STICKERS):
+            return cache_utility.EmptyCacheView()
```

Comment:
Missing test-case

You can do this in `tests.hikari.impl.test_cache.test_function_default`